### ISSUE TITLE
Invalid 0 precision token handling /get_tokens

### ISSUE THREAD

**Issue by Avm07** [eosrio/hyperion-history-api#131]
For 0 precision tokens missing precision field at /get_tokens response data

Example:
```
curl -X 'GET' \
  'https://hyperion.paycash.online/v2/state/get_tokens?account=i.list' \
  -H 'accept: application/json'
```

Logical estimate that field will be present:
```
{
      "symbol": "LQAG",
      "precision": 0,
      "amount": 0,
      "contract": "swap.pcash"
}

```

**Comment by chuck-h** [comment]
FYI @igorls looked at this a few months ago and thought it was ok
![image](https://user-images.githubusercontent.com/2141014/213260804-88dc4985-a175-4a2f-af06-c593bfe49bb0.png)

**Comment by chuck-h** [comment]
@Avm07 I agree with you it's not right and I'm glad to hear that @igorls agrees now too.

**Comment by igorls** [comment]
Thanks for opening the issue, we will get that patched soon

**Comment by igorls** [comment]
Fixed in commit 8036736 — 0-precision tokens now correctly return `precision: 0` instead of `undefined`. The token cache is also updated for 0-precision tokens.
